src/qmflows/packages/packages.py

```diff
@@ -432,38 +422,9 @@ def generic2specific(self, settings: Settings,
                     v)
                 continue
 
-            key = generic_dict.get(k)
-            if not key:
+            if not generic_dict.get(k):
                 self.handle_special_keywords(
                     specific_from_generic_settings, k, v, mol)
-                continue
```

Comment:
So uhm, what happened with all the code here?

Reply:
I am deleting that code because it is yet another `handle_special_keywords` mechanism that is not used by anyone.  I think that all the special keywords should be handled by the instances.

Reply:
That's fair, but is this already fully implemented?

Reply:
I have look for a piece of code that uses that snippet or in other libraries. But we are not using it anywhere. We also, handle the special keywords for those workflows that we are using, if needs rise to implement more handlers I will do so